### ISSUE TITLE
Permanent crashes on IntelliJ IDEA@2020.1.3 / import-cost@1.0.7.3

### ISSUE THREAD

**Issue by mdreizin** [denofevil/import-cost#28]
```
java.lang.NoSuchFieldError: myDefaultReporter
	at com.github.denofevil.importCost.LanguageService.createLanguageServiceQueue(LanguageService.kt:61)
	at com.github.denofevil.importCost.LanguageService.createLanguageServiceQueue(LanguageService.kt:34)
	at com.intellij.lang.javascript.service.JSAsyncLanguageServiceBase.computeProcessUnderLocks(JSAsyncLanguageServiceBase.java:246)
	at com.intellij.lang.javascript.service.JSAsyncLanguageServiceBase.lambda$getProcess$4(JSAsyncLanguageServiceBase.java:228)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:894)
	at com.intellij.openapi.application.ReadAction.compute(ReadAction.java:61)
	at com.intellij.lang.javascript.service.JSAsyncLanguageServiceBase.getProcess(JSAsyncLanguageServiceBase.java:226)
	at com.intellij.lang.javascript.service.JSLanguageServiceBase.getProcess(JSLanguageServiceBase.java:50)
	at com.intellij.lang.javascript.service.JSLanguageServiceBase.sendCommand(JSLanguageServiceBase.java:29)
	at com.github.denofevil.importCost.LanguageService.runRequest(LanguageService.kt:162)
	at com.github.denofevil.importCost.LanguageService.access$runRequest(LanguageService.kt:34)
	at com.github.denofevil.importCost.LanguageService$processImports$1$1.visitES6ImportDeclaration(LanguageService.kt:138)
	at com.intellij.lang.ecmascript6.psi.impl.ES6ImportDeclarationImpl.accept(ES6ImportDeclarationImpl.java:41)
	at com.intellij.psi.PsiWalkingState.visit(PsiWalkingState.java:70)
	at com.intellij.psi.PsiWalkingState.visit(PsiWalkingState.java:27)
	at com.intellij.util.WalkingState.walkChildren(WalkingState.java:65)
	at com.intellij.util.WalkingState.elementStarted(WalkingState.java:52)
	at com.intellij.psi.PsiWalkingState.elementStarted(PsiWalkingState.java:79)
	at com.intellij.lang.javascript.psi.JSRecursiveWalkingElementVisitor.visitElement(JSRecursiveWalkingElementVisitor.java:29)
	at com.intellij.psi.PsiElementVisitor.visitFile(PsiElementVisitor.java:35)
	at com.intellij.lang.javascript.psi.JSElementVisitor.visitJSFile(JSElementVisitor.java:327)
	at com.intellij.lang.javascript.psi.impl.JSFileImpl.accept(JSFileImpl.java:105)
	at com.github.denofevil.importCost.LanguageService$processImports$1.run(LanguageService.kt:123)
	at com.intellij.openapi.application.impl.ApplicationImpl.runReadAction(ApplicationImpl.java:869)
	at com.github.denofevil.importCost.LanguageService.processImports(LanguageService.kt:121)
	at com.github.denofevil.importCost.LanguageService.access$processImports(LanguageService.kt:34)
	at com.github.denofevil.importCost.LanguageService$updateImports$1.run(LanguageService.kt:111)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:333)
	at com.intellij.util.ui.update.MergingUpdateQueue.execute(MergingUpdateQueue.java:323)
	at com.intellij.util.ui.update.MergingUpdateQueue.lambda$flush$1(MergingUpdateQueue.java:273)
	at com.intellij.util.ui.update.MergingUpdateQueue.flush(MergingUpdateQueue.java:287)
	at com.intellij.util.ui.update.MergingUpdateQueue.run(MergingUpdateQueue.java:242)
	at com.intellij.util.concurrency.QueueProcessor.runSafely(QueueProcessor.java:232)
	at com.intellij.util.Alarm$Request.runSafely(Alarm.java:367)
	at com.intellij.util.Alarm$Request.run(Alarm.java:357)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at com.intellij.util.concurrency.SchedulingWrapper$MyScheduledFutureTask.run(SchedulingWrapper.java:220)
	at com.intellij.util.concurrency.BoundedTaskExecutor.doRun(BoundedTaskExecutor.java:222)
	at com.intellij.util.concurrency.BoundedTaskExecutor.access$200(BoundedTaskExecutor.java:29)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.execute(BoundedTaskExecutor.java:201)
	at com.intellij.util.ConcurrencyUtil.runUnderThreadName(ConcurrencyUtil.java:210)
	at com.intellij.util.concurrency.BoundedTaskExecutor$1.run(BoundedTaskExecutor.java:190)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
```

**Comment by mdreizin** [comment]
@denofevil Could you please take a look?

**Comment by anstarovoyt** [comment]
@mdreizin fixed via 1.0.7.4, sorry for this.

**Comment by mdreizin** [comment]
@anstarovoyt Thanks for fixing this.
